MixItUp.Base/Util/SpecialIdentifierStringBuilder.cs

```diff
@@ -460,6 +463,11 @@ public void ReplaceSpecialIdentifier(string identifier, string replacement)
             {
                 replacement = HttpUtility.UrlEncode(replacement);
             }
+            if (isFilePath)
```

Comment:
I don't fully understand why we're doing this? If the special identifier text has invalid characters in it, then that means its highly likely that it already isn't going to match up with an existing file path on the user's PC. So if we do any bad character replacement, its possible it won't match up with what the intention of what the user wants anyways. If they put a bad file path in and it doesn't work, it should just fail, IMO.

Reply:
Yeah, the problem is already an issue. For example, if we save a mixer clip using the $datetime, you convert invalid characters to underscores ( _ ), but then I can't turn around an "reference" that clip that was created because, the filename has invalid characters in it.

The code here is to allow us to specify that the special identifier is being used in a filename, and as such, it should have filename safe characters.  I'm not certain this is the "best" fix, if you don't like it, I'll just delete the PR.

Reply:
Well, I like using Special Identifiers for these areas, but I just think doing the character pathing might not be a good idea for right now. I guess it just seems more like a patch-work solution for this specific scenario than something that'll help in general in the longer-run.

For the Mixer Clips scenario, they could simply clip it to a set file name that will always be consistent (EX: lastreplay.mp4), then that solves the scenario. They don't get all their clips downloaded and will need to re-download them manually afterwards, but you can't always have your cake and eat it too. :)